css/index.css

```diff
@@ -101,10 +102,110 @@ p {
     justify-content: space-evenly;
 }
 
```

Comment:
This is where the cascade part of the CSS comes into play to your benefit.  In the HTML file.  the individual boxes were `<div class="box">Box1</div>`  if you added a class to it like `<div class="box box1">Box 1</div>` then you can now style everything in your individual boxes from the parent .box and just change the color of the box.  This would have saved 8 lines of code for every box.